CHANGELOG.md

```diff
@@ -26,6 +26,22 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.17.2...master
 
+### Added
+- Introduce `JqlRememberingProjectMemory` so that we have 2nd source of new JQL queries other than using `ViewIssueAction`, which already relies on `JqlMemory`.
+- Add `ProjectJqlFactory` and `SimpleProjectJqlFactory` primarily as components of the new `JqlRememberingProjectMemory`.
+- Introduce `LimitedJqlMemory` to allow for limiting number of queries remembered from specific source.
```

Comment:
Could you add a reason for adding LimitedJqlMemory here or to its javadoc? Why do we need to limit number of queries?

Reply:
The rationale is in [the commit message](https://github.com/atlassian/jira-actions/pull/108/commits/a398a9e1ce3ca6701516199d1cbf98994400bc53). I could add it to the CHANGELOG, however I was under impression that I'm already adding to much of context in there.